### PR TITLE
fix: fix proto wrappers debug impl

### PR DESCRIPTION
**Pull request by wyfo**
@milyin

### CODE REVIEW

File: commons/zenoh-config/src/wrappers.rs [hunk 1]
```diff
@@ -99,7 +103,13 @@ impl fmt::Display for Hello {
     }
 }
 
-#[derive(Debug, Default, Copy, Clone, Eq, Hash, PartialEq)]
+impl fmt::Display for Hello {
```
Comment (wyfo) [inline commons/zenoh-config/src/wrappers.rs:106]: @Mallets @milyin Is there a reason to have a `Display` implementation that looks like a `Debug` one on `Hello`.
When the previous `Hello` type contained the `version` in its `Debug` implementation, why not, but now, shouldn't we remove this `Display` implementation?

**Review by milyin — APPROVED** [review]

File: commons/zenoh-config/src/wrappers.rs (hunk 1 above)
Comment (Mallets) [reply commons/zenoh-config/src/wrappers.rs:106]: I would keep `Debug` printing all information and the `Display` a simplified version of it.

Comment (Mallets) [reply commons/zenoh-config/src/wrappers.rs:106]: To me the `Debug` should print the whole internal state.

Comment (wyfo) [reply commons/zenoh-config/src/wrappers.rs:106]: Ok, so I will fix the `Debug` impl.

**Pull request merged**